apps/studio/components/interfaces/Integrations/IntegrationConnection.tsx

```diff
@@ -49,6 +44,8 @@ const IntegrationConnectionItem = forwardRef<HTMLLIElement, IntegrationConnectio
     const onConfirm = useCallback(async () => {
       try {
         await onDeleteConnection(connection)
+      } catch (error) {
+        // [Joshen] No need for error handler
```

Comment:
In 2 of the 3 uses of the this component, there's a throw in the promise.
https://github.com/supabase/supabase/blob/master/apps/studio/components/interfaces/Organization/IntegrationSettings/IntegrationSettings.tsx#L61
https://github.com/supabase/supabase/blob/master/apps/studio/components/interfaces/Settings/Integrations/GithubIntegration/GithubSection.tsx#L79

Reply:
ahhhh good catch - they shouldn't need to throw, i'll just make them return (like how we handle projectref checks before calling a mutation)

Reply:
okay pushed a change! 🙏